### ISSUE TITLE
@tsed/socketio@5.2.0 is not backwards compatible with @tsed/socketio@5.1.3

### ISSUE THREAD

**Issue by kpapadatos** [Romakita/ts-express-decorators#517]
## Information

- **Version:** @tsed/socketio@5.2.0
- **Type:** Issue

## Description
I had `@tsed/socketio@5.1.3` in my `package.json` with a `^` as installed by default (`"@tsed/socketio": "^5.1.3"`). Today, after doing `npm i` on a new `git clone`, the project errored when `socketIO: {}` was provided in the `@tsed` `Server` decorator configuration. Not sure if this is a bug with version 5.2.0 or just a compatibility issue.

Error was something like `Cannot add path because object is not extensible`.

## Acceptance criteria

- [ ] Investigate and fix if this is a bug.
- [ ] Mark `@tsed/socketio@5.2.0` as incompatible with`@tsed/socketio@5.1.3`.

**Comment by Romakita** [comment]
Hum can you send me your ServerSettings configuration for socketIo ? pls :)

**Comment by kpapadatos** [comment]
@Romakita `@tsed/socketio@5.1.3` this works:
```js
socketIO: { adapter: redisAdapter(redisConfig) }
```

With `@tsed/socketio@5.2.0` it errors. The following errors too:
```js
socketIO: {
   // adapter: redisAdapter(redisConfig)
}
``` 

It only works if I remove the `socketIO` config entirely like so:
```js
// socketIO: {},
```

Switching back to 5.1.3 fixes it.

**Comment by Romakita** [comment]
Fixed :D v5.2.1
